### ISSUE TITLE
How can i embed videos?

### ISSUE THREAD

**Issue by iL-0** [notaapp/issues#435]
Hello,
Is there any way to embed local file (video) or web video such as Youtube to Nota?
Thanks :)

**Comment by astoilkov** [comment]
No, it's currently not possible. Are you using something that supports that?

**Comment by iL-0** [comment]
No, I was just curious if it’s possible.
It would be cool feature.
